Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -714,6 +714,8 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver,
             ongoingVideoCallControls.removeFromSuperview()
             ongoingCallControls.insertArrangedSubview(ongoingAudioCallControls, at: 0)
         }
+        // Layout immediately to avoid spurious animation.
+        ongoingCallControls.layoutIfNeeded()
```

Comment:
The repro is to enable then disable local animations. When disabling, there was an undesired animation.

Reply:
LGTM